src/components/predictive-layer.vue

```diff
@@ -14,8 +14,22 @@
           : 'x-row x-row--gap-06 x-row--align-start'
```

Comment:
Not for this PR, but I would try to improve the readability of this template using grouped ifs, and extracting some components. It is starting to be a mess.

Reply:
Right, not in this PR